packages/core/src/backend.ts

```diff
@@ -804,7 +805,11 @@ export class Backend<EF extends BaseField = UnknownField, BC extends BackendClas
             entry,
             undefined,
           );
-          return this.implementation.getMedia(folder, mediaPath);
+          return this.implementation.getMedia(
+            folder,
+            mediaPath,
+            configState.config!.media_library_folder_support,
```

Comment:
Instead of asserting that it won't be null, I would prefer we provide a default instead.
```suggestion
            configState.config.media_library_folder_support ?? false,
```

Reply:
Resolved with last commit